core/sass/partials/_effects.scss

```diff
@@ -1,189 +0,0 @@
-// EFFECT STYLES
```

Comment:
moved this styling into `_carousel` and `_modal` to make it more clear from the naming what it's for